docs: add Stanimira Vlaeva to contributors

sis0k0 opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?

This PR adds [Stanimira Vlaeva](https://twitter.com/StanimiraVlaeva) to the list of Angular GDEs. -> cc @StephenFluin


## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

sis0k0 committed:
docs: add Stanimira Vlaeva to contributors

StephenFluin commented on aio/content/marketing/contributors.json:
```diff
@@ -699,12 +699,22 @@
     "bio": "Brian is a software engineer and GDE in Angular with a passion for learning, writing, speaking, teaching and mentoring. Brian has been building web applications for over 20 years and has long been a fanboy of JavaScript. When not in front of his Macbook Pro Brian is in the Rocky Mountains skiing or hiking.",
     "group": "GDE"
   },
+
```
Can we remove this extra whitespace change?

gkalpak replied:
Whyyyyy? 😢 It is making whitespace consistent with the rest of the file. 👌 
I whole:heart:edly approve of this whitespace change (and frown upon @jeffbcross's introducing the inconsistency in the first place 👀)

gkalpak replied:
Ideally, we should get @jeffbcross to submit a PR with the fix 🦈

StephenFluin replied:
I can be onboard with fixing it here. I just have an initial allergic reaction to unrelated changes. :)

jeffbcross replied:
How dare you question my whitespace judgment, @gkalpak

gkalpak replied:
Can't read your comment. The test is blurry. I think it's because I'm still frowning at you 😒

brandonroberts commented:
Caretaker: adding a global approval

IgorMinar committed:
style: reformat aio/content/marketing/contributors.json with clang-format

without this change, it's difficult to edit this file for anyone who has IDE configured for angular/angular development.
